packages/react-native-usb/android/src/main/java/io/trezor/rnusb/ReactNativeUsbModule.kt

```diff
@@ -264,8 +267,7 @@ class ReactNativeUsbModule : Module() {
 
     private fun closeAllOpenedDevices() {
         Log.d(LOG_TAG, "Closing all devices")
-        pendingRequests.values.forEach { it.cancel() }
-        pendingRequests.clear()
```

Comment:
Removing this will break THP. We'll have to figure out how ensure both use cases work correctly.

Reply:
yes 🙉